contrib/gce/configure.sh

```diff
@@ -235,6 +238,22 @@ disabled_plugins = ["io.containerd.internal.v1.restart"]
 EOF
 chmod 644 "${config_path}"
 
+
+docker_registry_host_namespace="${registry_config_path}/docker.io/hosts.toml"
+mkdir -p $(dirname ${docker_registry_host_namespace})
+cat > ${config_path} <<EOF
+server = "https://docker.io"
+
+[host."https://mirror.gcr.io"]
+  capabilities = ["pull"]
```

Comment:
Use `capabilities = ["pull", "resolve"]`, otherwise the next host will be used to resolve tags.